src/main/java/com/google/api/codegen/go/GoGapicContext.java

```diff
@@ -544,7 +547,7 @@ public boolean hasEmptyRetryCodes(Interface service, MethodConfig methodConfig)
    * Returns true if the API has a page streaming method.
    */
   public boolean hasPageStreamingMethod() {
-    for (Interface service : getModel().getSymbolTable().getInterfaces()) {
+    for (Interface service : new InterfaceView().getElementIterable(getModel())) {
```

Comment:
It would be nice if the `new InterfaceView().getElementIterable(model)` pattern could be a method somewhere, but I'm not sure if there is anywhere sensible...